addons/mail/models/res_partner.py

```diff
@@ -214,7 +223,7 @@ def _find_or_create_from_emails(self, emails, additional_values=None):
 
     def _to_store(self, store: Store, /, *, fields=None, main_user_by_partner=None):
         if fields is None:
-            fields = ["active", "email", "im_status", "is_company", "name", "user", "write_date"]
+            fields = ["active", "email", "im_status", "is_company", "name", "user", "write_date", "custom_status"]
```

Comment:
Is there a reason to set `forced_status` to `im_status` but not `custom_status`? I feel like we could just return `im_status` whatever it is.

Reply:
The discussion with Didier came from a misunderstanding of `custom_status`.

`custom_status` has actually nothing to do with the `im_status`, it's the little message the user can customize that shows up next to their user name (e.g. OoO until blabla).